### PR TITLE
refactor: Updated Navigation component and added footer

### PR DESCRIPTION
**Pull request by sharath-io**
Updated navigation component and added footer
deploy preview: https://deploy-preview-9--atom-ui.netlify.app

Screen Shot 2022-02-23 at 1 45 28 AM
Screen Shot 2022-02-23 at 1 45 36 AM

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (sah-shah7) [inline .vscode/settings.json:2]: You can delete this

**Review by sah-shah7 — APPROVED** [review]
Just 1 change, rest is okay to merge

File: .vscode/settings.json (hunk 1 above)
Comment (sharath-io) [reply .vscode/settings.json:2]: This came through vs code. WIll delete this.

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

**Pull request merged**